controllers/gcpmachine_controller.go

```diff
@@ -50,48 +51,49 @@ type GCPMachineReconciler struct {
 	client.Client
 	Log              logr.Logger
```

Comment:
Similar to the cluster controller, if we are using the logger from the context, I think this should probably be removed here.